internal/commands/ip_access/create_inputs.go

```diff
@@ -0,0 +1,22 @@
+package ipaccess
+
+import (
+	"github.com/10gen/realm-cli/internal/cli"
+	"github.com/10gen/realm-cli/internal/cli/user"
+	"github.com/10gen/realm-cli/internal/terminal"
+)
+
+type createInputs struct {
+	cli.ProjectInputs
+	IPAddress  string
+	Comment    string
+	UseCurrent bool
+	AllowAll   bool
+}
+
+func (i *createInputs) Resolve(profile *user.Profile, ui terminal.UI) error {
```

Comment:
the separate files for these inputs seems a bit overkill, let's consolidate all things "create" related to here to the `create.go` file